ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/AttributeAnalyzerTest/JavaAttributeVisitorTest.java

```diff
@@ -0,0 +1,238 @@
+package edu.usb.argos.ASTProcessor.AttributeAnalyzerTest;
+
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.AttributeHandler;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInfo;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.JavaAttributeVisitor;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+
+class JavaAttributeVisitorTest {
+
+    private static JavaAttributeVisitor visitor;
+
+    @BeforeAll
+    public static void init() {
+        AttributeHandler attributeHandler = new AttributeHandler();
+        visitor = new JavaAttributeVisitor(attributeHandler);
+    }
+
+    private Optional<JavaParser.ClassBodyContext> getClassFromText(String path) {
+        CharStream input = CharStreams.fromString(path);
+
+        JavaLexer lexer = new JavaLexer(input);
+        CommonTokenStream tokens = new CommonTokenStream(lexer);
+        JavaParser parser = new JavaParser(tokens);
+
+        JavaParser.CompilationUnitContext compilationUnitContext = parser.compilationUnit();
+
+        if (!compilationUnitContext.typeDeclaration().isEmpty()) {
+            JavaParser.TypeDeclarationContext typeDecl = compilationUnitContext.typeDeclaration().get(0);
+            if (typeDecl.classDeclaration() != null) {
+                return Optional.ofNullable(typeDecl.classDeclaration().classBody());
+            }
+        }
+
+        return Optional.empty();
+    }
+
+    @Test
+    void testVisitAttribute_withModifiers() {
```

Comment:
Can you explain me a little bit why you used the snake convention here and others below?

Reply:
It was to make it easier to understand the unit test case, but you are right.